lib/game.rb

```diff
@@ -0,0 +1,106 @@
+require_relative "./board.rb"
+require_relative "./player.rb"
+require_relative "players/human.rb"
+require_relative "players/computer.rb"
+
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ]
+
+  def initialize(player_1=Human.new("X"), player_2=Human.new("O"), board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+
+  def current_player
+    current_player = @player_1
+    if board.turn_count%2!=0
+      current_player = @player_2
+    end
+    return current_player
+  end
```

Comment:
Use [`Integer#even?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-even-3F) or [`Integer#odd?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-odd-3F)

```
if board.turn_count.odd? 
  current_player = @player_2
end
```
You could further simplify with the ternary operator:
```
current_player = (board.turn_count.even? ? @player_1 : @player_2)
return current_player
```
Finally, you could use the implicit return value of the expression to remove the local variable `current_player`
```
def current_player
  board.turn_count.even? ? @player_1 : @player_2
end
```